gwlfe/gwlfe.py

```diff
@@ -16,7 +16,7 @@
 from . import WriteOutputFiles
 
 
-def run():
+def run(z):
```

Comment:
Same with `z`, was it just picked because it's short and unlikely to clash with anything meaningful?  I think I'm into it, as you wouldn't want to see something like `data.***********` everywhere in the code.